src/Uno.UWP/UI/Input/GestureRecognizer.Gesture.cs

```diff
@@ -46,11 +45,17 @@ private class Gesture
 
 			public bool HasExceedMinHoldPressure { get; private set; }
 
+			internal GestureSettings Settings
+			{
+				get;
+				private set;
+			}
```

Comment:
I think we don't need to make it externally visible anymore, I would keep it as a private field.

Reply:
We still need to return Gesture.Settings instead of GestureRecognizer.Settings in PreventEvents as Gesture.Settings can be overriden inside Gesture